CHANGELOG.md

```diff
@@ -192,7 +192,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
-- The `llvm-tools` component was renamed to `llvm-tools-preview`
+- The `llvm-tools` component was renamed to `llvm-tools`
```

Comment:
You need to add new line under `Unreleased`, not here.

Reply:
Fixed